zerver/models.py

```diff
@@ -1567,6 +1568,7 @@ class UserBaseSettings(models.Model):
         left_side_userlist=bool,
         starred_message_counts=bool,
         translate_emoticons=bool,
+        display_emoji_reaction_users=bool,
```

Comment:
This was incorrectly added to `display_settings_legacy` ... which makes me think that we've not set ourselves up for success here. This should have been directly added to  `property_types`, below. Would it have helped if we have a `modern_settings` list above these two `_legacy` lists (containing the handful of "modern" settings) and perhaps a comment stating that nothing new should be added to these legacy structures?

@laurynmm @sahil839 FYI for thoughts on this detail as well.

Reply:
Yes, a comment would be helpful.